includes/class-wc-cart-session.php

```diff
@@ -239,4 +236,125 @@ private function set_cart_cookies( $set = true ) {
+	 *
+	 * @access private
+	 * @since  3.5.0
+	 * @param int   $order_id Order ID to try to load.
+	 * @param array $cart Current cart array.
+	 * @return array
+	 */
+	private function populate_cart_from_order( $order_id, $cart ) {
+		$order = wc_get_order( $order_id );
+
+		if ( ! $order->get_id() || ! $order->has_status( apply_filters( 'woocommerce_valid_order_statuses_for_order_again', array( 'completed' ) ) ) || ! current_user_can( 'order_again', $order->get_id() ) ) {
+			return;
+		}
+
+		if ( apply_filters( 'woocommerce_empty_cart_when_order_again', true ) ) {
+			$cart = array();
+		}
+
+		$inital_cart_size = count( $cart );
+		$order_items      = $order->get_items();
+
+		foreach ( $order_items as $item ) {
+			$product_id     = (int) apply_filters( 'woocommerce_add_to_cart_product_id', $item->get_product_id() );
+			$quantity       = $item->get_quantity();
+			$variation_id   = (int) $item->get_variation_id();
+			$variations     = array();
+			$cart_item_data = apply_filters( 'woocommerce_order_again_cart_item_data', array(), $item, $order );
+			$product        = $item->get_product();
+
+			if ( ! $product ) {
+				continue;
+			}
+
+			// Prevent reordering variable products if no selected variation.
+			if ( ! $variation_id && $product->is_type( 'variable' ) ) {
+				continue;
+			}
+
+			foreach ( $item->get_meta_data() as $meta ) {
+				if ( taxonomy_is_product_attribute( $meta->key ) ) {
+					$term                     = get_term_by( 'slug', $meta->value, $meta->key );
+					$variations[ $meta->key ] = $term ? $term->name : $meta->value;
+				} elseif ( meta_is_product_attribute( $meta->key, $meta->value, $product_id ) ) {
+					$variations[ $meta->key ] = $meta->value;
+				}
+			}
+
+			if ( ! apply_filters( 'woocommerce_add_to_cart_validation', true, $product_id, $quantity, $variation_id, $variations, $cart_item_data ) ) {
+				continue;
+			}
+
+			// Add to cart directly.
+			$cart_id          = WC()->cart->generate_cart_id( $product_id, $variation_id, $variations, $cart_item_data );
+			$product_data     = wc_get_product( $variation_id ? $variation_id : $product_id );
+			$cart[ $cart_id ] = apply_filters(
+				'woocommerce_add_cart_item', array_merge(
+					$cart_item_data, array(
+						'key'          => $cart_id,
+						'product_id'   => $product_id,
+						'variation_id' => $variation_id,
+						'variation'    => $variations,
+						'quantity'     => $quantity,
+						'data'         => $product_data,
+						'data_hash'    => wc_get_cart_item_data_hash( $product_data ),
+					)
+				), $cart_id
+			);
+		}
+
+		do_action( 'woocommerce_ordered_again', $order->get_id() );
+
+		$num_items_in_cart           = count( $cart );
+		$num_items_in_original_order = count( $order_items );
+		$num_items_added             = $num_items_in_cart - $inital_cart_size;
+
+		if ( $num_items_in_original_order > $num_items_added ) {
+			wc_add_notice(
+				sprintf(
+					// Translators: %d item count.
```

Comment:
WordPress standard for this is:

```php
/* translators: %d item count */
```

Reply:
You can found it in https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#descriptions